doc/queries.md

```diff
@@ -556,15 +559,63 @@ Result:
 <a name="filter"></a>
 
 ## Filters 
-Currently Guppy uses `JSON`-based syntax for filters. The JSON object key could be an operation like `=`, `>`. One simple example could be:
+
+<a name="filter-unit"></a>
+
+### Basic filter unit
+Currently Guppy uses `JSON`-based syntax for filters.
+The JSON object key could be an operation like `=`, `>`. 
+A very basic filter unit would look like: `{<operater>: {<field_name> : <value_expression>}}`. 
+One simple example could look like:
 
 ```
 {
-  "filter": {"=": {"subject_id": "69"}}
+  "filter": {
+    "=": {
+      "subject_id": "sbj_69"
+    }
+  }
 }
 ```
 
-Or you could use binary combination (`AND` or `OR`)to combine simple filter units into more complicated big filters. Example:
+
+Currently we support following operators: 
+
+
+| operator     | meaning                  | support field type | example                                                          |
+|--------------|--------------------------|--------------------|------------------------------------------------------------------|
+| eq, EQ, =    | equal                    | string, number     | {"eq": {"gender": "female"}}                                     |
+| in, IN       | inside                   | string, number     | {"in": {"gender": ["female", "F"]}}                              |
+| !=           | is not                   | string, number     | {"!=": {"gender": "male"}}                                       |
+| gt, GT, >    | greater than             | number             | {">": {"age": 50}}                                               |
+| gte, GTE, >= | greater than or equal to | number             | {">=": {"age": 50}}                                              |
+| lt, LT, <    | less then                | number             | {"<": {"age": 50}}                                               |
+| lte, LTE, <= | less than or equal to    | number             | {"<=": {"age": 50}}                                              |
+| search       | [search text](#filter-search)              | text               | {"search": {"keyword": "asian","fields": ["race", "ethnicity"]}} |
+
+
+
+<a name="filter-search"></a>
```

Comment:
Minor comment: you don't need to put the anchors in Markdown manually, all heading on Github already have them. See more [here](https://gist.github.com/asabaylus/3071099#gistcomment-1593627).

Reply:
ah good to know! thanks!